components/TripMapComponent.tsx

```diff
@@ -39,14 +39,15 @@ const TripMapComponent = (props: ConnectedProps<typeof connector> & TripMapCompo
 
   return <>
     <MapView ref={mapRef} onLayout={l => setMapLayout(l.nativeEvent.layout)}
-      mapType="standard"
       provider="google"
```

Comment:
Hvorfor er mapType fjernet?

Reply:
Default er "standard"

Reply:
for å få kartene like. Standard er standard